### PR TITLE
Update contributor guide

### PR DESCRIPTION
**Pull request by mortent**
Update the contributor guide with better guidance for anyone who wants to contribute to Config Sync.

Fixes: https://github.com/GoogleContainerTools/kpt-config-sync/issues/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,5 +15,14 @@ Cloud CLI.
 We welcome contributions to Config Sync from the community. Take a look at our
 [contribution guide] to get started.
 
+## Community, discussion and support
+You can reach the maintainers of this project at:
+
+* [github.com/GoogleContainerTools/kpt-config-sync]
```
Comment (mikebz) [inline README.md:21]: Do we want to emphasize the discussion groups?

File: docs/contributing.md [hunk 2]
```diff
@@ -1,11 +1,22 @@
 # How to Contribute
 
-ConfigSync follows the [Kubernetes contribution process].  If you
-are looking to contribute to this project please fork it, create a change
-and then submit a PR against the `main` branch which is the HEAD of development.
+Contributions are greatly appreciated. The maintainers of Config Sync monitor issues
+and PRs and strive to provide timely feedback and reviews. For any questions or 
+live discussions please ask questions in the #kpt channel on [slack].
```
Comment (mikebz) [inline docs/contributing.md:5]: I think @janetkuo mentioned that she would prefer async comms on discussion groups or issues.  Is that something we want to state?

**Comment by mikebz** [comment]
Linking with #18

File: README.md (hunk 1 above)
Comment (mortent) [reply README.md]: Added the discussion group to the top of the list

File: README.md [hunk 3]
```diff
@@ -15,5 +15,15 @@ Cloud CLI.
 We welcome contributions to Config Sync from the community. Take a look at our
 [contribution guide] to get started.
 
+## Community, discussion and support
+You can reach the maintainers of this project at:
+
+* [github.com/GoogleContainerTools/kpt-config-sync discussions]
+* [github.com/GoogleContainerTools/kpt-config-sync issues]
```
Comment (haiyanmeng) [inline README.md:22]: These two links are broken with `404` errors.

File: docs/contributing.md [hunk 4]
```diff
@@ -1,11 +1,23 @@
 # How to Contribute
 
-ConfigSync follows the [Kubernetes contribution process].  If you
-are looking to contribute to this project please fork it, create a change
-and then submit a PR against the `main` branch which is the HEAD of development.
+Contributions are greatly appreciated. The maintainers of Config Sync monitor issues
+and PRs and strive to provide timely feedback and reviews. For any questions or 
+suggestions, reach to to the Config Sync maintainers on [github.com/GoogleContainerTools/kpt-config-sync issues]
+or [github.com/GoogleContainerTools/kpt-config-sync discussions].
```
Comment (haiyanmeng) [inline docs/contributing.md:6]: These two links are broken with `404` errors.

**Pull request merged**